tests/snapshots/init/ignores_failure_to_format_source.in/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+tab_spaces = "broken toml config"
+@*&~~borken~~~~syntax~~?
```

Comment:
Is this helping verify this test case (rustfmt not in PATH) or is this another independent ignore-failure test case?

Reply:
Nope. I should remove this line and change to `tab_spaces = 2`. Then we can observe that rustfmt doesn't take any effect.